hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/ServiceCallViewGenerator.java

```diff
@@ -320,13 +322,11 @@ private void buildEntrySpanView(
 
     // If this is a HTTP request, parse the http response status code.
     if (protocol == Protocol.PROTOCOL_HTTP || protocol == Protocol.PROTOCOL_HTTPS) {
+      Optional<Request> httpRequest = Optional.ofNullable(event.getHttp())
```

Comment:
Note: I think, we should add tests for all views too (not a blocker for this PR)